README.md

```diff
@@ -110,12 +110,12 @@ export $(cat .env | xargs) # Only run this if you want to set or update the curr
 ```
 # Import the supplier price spreadsheet that has been uploaded to S3
 
-java -jar app.jar --spring.main.web-application-type=none --price-import --supplier SERCO/GEOAMEY --year 2020
+java -jar app.jar --spring.main.web-application-type=none --price-import --supplier=SERCO/GEOAMEY --year=2020
```

Comment:
Noticed an error in the readme for running the commands.  No related to this PR fix.